src/main/resources/org/jenkinsci/plugins/stashNotifier/StashNotifier/help-projectKey.html

```diff
@@ -0,0 +1,3 @@
+<div>
+  If not empty, this key will be used instead of automatically configured project key
```

Comment:
I propose to mention that the value of the setting _Prepend parent project key_ is ignored if an override project key is specified.